src/dotnet/APIView/APIViewWeb/Controllers/AutoReviewController.cs

```diff
@@ -40,5 +44,52 @@ public async Task<ActionResult> UploadAutoReview([FromForm] IFormFile file, stri
             // Return internal server error for any unknown error
             return StatusCode(statusCode: StatusCodes.Status500InternalServerError);
         }
+
+        [HttpGet]
+        public async Task<ActionResult> GetReviewStatus(string language, string packageName)
+        {
+            // This API is used by prepare release script to check if API review for a package is approved or not.
+            // This caller script doesn't have artifact to submit and so it can't check using create review API
+            // So it rely on approval status of latest revision of automatic review for the package
+            // With new restriction of creating automatic review only from master branch or GA version, this should ensure latest revision
+            // is infact the version intended to be released.
+
+            TelemetryClient telemetryClient = new TelemetryClient(TelemetryConfiguration.CreateDefault());
+            var requestTelemetry = new RequestTelemetry { Name = "Checking review status for package " + packageName };
+            var operation = telemetryClient.StartOperation(requestTelemetry);
```

Comment:
We don't need the `RequestTelemetry` here. It would be created for the incoming HTTP request.